src/ucm/malloc/malloc_hook.c

```diff
@@ -600,6 +600,26 @@ static void ucm_malloc_install_optional_symbols()
     }
 }
 
+static void ucm_malloc_install_mallopt()
+{
+    /* copy values of M_MMAP_THRESHOLD and M_TRIM_THRESHOLD
+     * if they were overriden by the user
+     */
+    char *p;
+
+    p = getenv("MALLOC_TRIM_THRESHOLD_");
+    if (p) {
+        ucs_debug("set trim_thresh to %d", atoi(p));
```

Comment:
i would use alol() and 'long' instead of atoi

Reply:
mallopt() arguments are integers

Reply:
ok